airflow/www/templates/airflow/datasets.html

```diff
@@ -22,7 +22,8 @@
 
 {% block head_meta %}
   {{ super() }}
-  <meta name="datasets_api" content="{{ url_for('/api/v1.airflow_api_connexion_endpoints_dataset_endpoint_get_datasets') }}">
+  <meta name="datasets_api" content="{{ url_for('Airflow.get_datasets') }}">
```

Comment:
```suggestion
  <meta name="datasets_api" content="{{ url_for('Airflow.datasets_summary') }}">
```

Need to update this to match the function name in `views.py`